src/pages/graphql/schema/store/queries/countries.md

```diff
@@ -12,6 +12,10 @@ Use the [country](country.md) query if you want to retrieve information about a
 
 `{countries {Countries}}`
 
+## Reference
+
+The [`countries`](https://developer.adobe.com/commerce/webapi/graphql-api/beta/index.html#query-countries) reference provides detailed information about the types and fields defined in this query.
```

Comment:
should this link to the `/beta` or https://developer.adobe.com/commerce/webapi/graphql-api/index.html#query-countries?

Reply:
Thanks for catching this. I've removed `beta` from all URLs in this PR.